### DISCUSSION TITLE
Distributed mode implementation

### DISCUSSION THREAD

**Discussion by Ukd1796** [alibaba/FederatedScope#531] [Q&A]

**Comment by xieyxclack** [comment]
Hi, @Ukd1796 , you can refer to [here](https://github.com/alibaba/FederatedScope/tree/master/scripts#distribute-mode) the provided example of running FS with distributed mode. 

- First of all, please specify the ip addresses (i.e., `server_host`, `server_port`, `client_host`, `client_port`) when running FS with multiple devices.
- Then, if the provided example can be successfully run, you can use your own data/model as that in standalone mode. (please refer to the [tutorial](https://federatedscope.io/docs/own-case/) .
- And please double check the network connection if the provided example could not be run.


Thanks for your interest on FederatdedScope!  Please feel free to discuss here if you have more questions.
